SampleCommunicator/build.gradle

```diff
@@ -45,7 +50,7 @@ dependencies {
 
     // Logging
     implementation group: 'org.slf4j', name: 'slf4j-api', version: '2.0.10'
-    implementation group: 'org.apache.logging.log4j', name: 'log4j-slf4j2-impl', version: '2.23.1'
+    implementation group: 'org.apache.logging.log4j', name: 'log4j-slf4j2-impl', version: '2.19.0'
```

Comment:
Ich musste zurück auf 2.23.1 ansonsten habe ich keinen logging output.  Ist das bei Dir gerade umgekehrt, geht's nur mit 2.19.0?

Reply:
Bei mir läuft's auch mit 2.19.0 (frisch ausgeheckt auf meinem Mac zuhause). Läuft aber auch mit 2.23.1.
Ich glaube, ich habe die Versionen an SGrJava angeglichen.

Reply:
OK, I will try to find out what's wrong on my side...

Reply:
Ich hab's nun angeschaut.

Ich habe mit 2.19.0 folgende Fehlermeldung:
```
ERROR StatusLogger Log4j2 could not find a logging implementation. Please add log4j-core to the classpath. Using SimpleLogger to log to the console...
17:52:57: Execution finished ':BasicSampleCommunicator.main()'.
```

Ich kriege es zum Laufen wenn ich log4j-core hinzfüge.
```
// Logging
    implementation group: 'org.slf4j', name: 'slf4j-api', version: '2.0.10'
    implementation group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.19.0'
    implementation group: 'org.apache.logging.log4j', name: 'log4j-slf4j2-impl', version: '2.19.0'
```

Ich schlage vor wir fügen den core hinzu um sicherzugehen...

Reply:
Das erklärt aber noch nicht wieso es bei Dir ohne core mit 19 läuft und bei mir nur mit 23....

Reply:
Ja, das ist merkwürdig. Ich stimme zu, den core hinzuzufügen, dann sollte es ja passen.